code/datums/gamemodes/gangwar.dm

```diff
@@ -1040,56 +1579,180 @@ proc/broadcast_to_all_gangs(var/message)
 		//cash score
 		if (istype(item, /obj/item/currency/spacecash))
 			var/obj/item/currency/spacecash/S = item
-			if (S.amount > 500)
-				boutput(user, SPAN_ALERT("<b>[src.name] beeps, it don't accept bills larger than 500[CREDIT_SIGN]!</b>"))
-				return 0
 
-			gang.score_cash += round(S.amount/CASH_DIVISOR)
-			gang.spendable_points += round(S.amount/CASH_DIVISOR)
+			var/cash_to_take = max(0,min(GANG_LAUNDER_CAP-stored_cash, S.amount))
+
+			if (S.hasStatus("freshly_laundered"))
+				superlaunder_stacks += round(cash_to_take/(GANG_LAUNDER_RATE*1.5))
+
+			if (cash_to_take == 0)
+				boutput(user, SPAN_ALERT("<b>You've crammed the money laundering slot full! Let it launder some.<b>"))
+				return
+			if (stored_cash == 0)
+				boutput(user, SPAN_ALERT("The [src] boots up and starts laundering the money. This will take some time, so defend it!"))
+			if (cash_to_take < S.amount)
+				stored_cash += cash_to_take
+				S.amount -= cash_to_take
+				boutput(user, SPAN_ALERT("<b>You load [cash_to_take][CREDIT_SIGN] into the [src.name], the laundering slot is full.<b>"))
+				S.UpdateStackAppearance()
+				return
+			stored_cash += S.amount
 
 		//gun score
 		else if (istype(item, /obj/item/gun))
 			if(istype(item, /obj/item/gun/kinetic/foamdartgun))
 				boutput(user, SPAN_ALERT("<b>You cant stash toy guns in the locker</b>"))
 
-				return 0
-			// var/obj/item/gun/gun = item
-			gang.score_gun += round(300)
-			gang.spendable_points += round(300)
+				return
 
+			if(istype(item, /obj/item/gun/kinetic/slamgun))
+				boutput(user, SPAN_ALERT("<b>You cant stash toy guns in the locker</b>"))
```

Comment:
```suggestion
				boutput(user, SPAN_ALERT("<b>This shoddy firearm is worth a lot less</b>"))
```
Also maybe include zipguns if you want to make makeshift stuff worth less?

Reply:
nah, zipguns require staplers which are a bit more annoying to get ahold of (and have staffie charm)